src/runtime/propensity-server.js

```diff
@@ -116,7 +150,7 @@ export class PropensityServer {
     if (context) {
       eventInfo = eventInfo + ':' + encodeURIComponent(context);
     }
-    let url = ServiceUrl.adsUrl('/subopt/data?events=')
+    let url = adsUrl('/subopt/data?events=')
```

Comment:
Line 143: Any reason this method should stay public?

if the only caller is from within we can rename this to sendEvent_() and make it private.

Reply:
Agreed